src/frontends/pytorch/src/op/reshape.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_reshape(NodeContext& context) {
+    auto shape_node = context.get_input(1).get_node();
+    auto shape_node_fw_node = dynamic_cast<PtFrameworkNode*>(shape_node);
+    std::shared_ptr<ov::Node> reshape;
+    // TODO: move this to transform stage
+    if (shape_node_fw_node && shape_node_fw_node->get_decoder()->get_op_type() == "prim::ListConstruct") {
```

Comment:
I was thinking you will move this to transformation step, not copy from `aten::view`

Reply:
As we discussed I will add this transformation as soon as other required operations will be ready.